src/pyflexplot/data/presets/opr/cosmo-1e-ctrl/all_pdf.toml

```diff
@@ -1,7 +1,7 @@
 # PyFlexPlot setup file to create deterministic COSMO-1E-CTRL plots
 
 [_base]
-# Data source: tsa:/scratch/ruestefa/shared/test/pyflexplot/
+# Test data source: /store/mch/msopr/pyflexplot_testdata/
```

Comment:
@pirmink @twicki @petrabaumann @DanielLeuenberger We should review the path for testdata after /store on balfrin has been finalized. Basically, I would prefere a common directory for all testdata with subdirectories for the different applications. Nexus could also be an option.